website/project/metadata/brandt-postcomp-2.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
Eric has made some changes to schemas, such as making `queestions` a list instead of an object (in order to preserve field and question order). Am I correct in assuming that those changes are independent of this PR and not in the scope of draft reg?